doc/rst/source/explain_3D_symbols.rst_

```diff
@@ -0,0 +1,31 @@
+    All the previous symbols are 2-D and are plotted in the *x*-*y* plane only.
```

Comment:
This file is not included in plot3d.

Reply:
In plot3d_common.rst_
```
.. _-S:

.. include:: explain_symbols.rst_

.. include:: explain_3D_symbols.rst_

```

Reply:
OK, I must have missed it in my previous build.